src/AccessibilityInsights.SharedUx/Dialogs/ElementInfoDialog.xaml

```diff
@@ -96,7 +96,7 @@
                     <behaviors:ExpanderBehavior/>
                 </i:Interaction.Behaviors>
                 <Expander.Header>
-                    <Label  Margin="4" Style="{StaticResource ResourceKey=lblAccordionHeader}" x:Name="labalFixFollowing" Content="{x:Static Properties:Resources.labalFixFollowingContent2}" />
+                    <Label  Margin="4" Style="{StaticResource ResourceKey=lblAccordionHeader}" x:Name="labelFixFollowing" Content="{x:Static Properties:Resources.labelFixFollowingContent2}" />
```

Comment:
Looks like you changed the typos for the content resource name here and in the other 2 files but didn't make the analogous changes for typos in the resource files

Reply:
@jalkire, you need to load the diff for src/AccessibilityInsights.SharedUx/Properties/Resources.Designer.cs to see the corresponding changes. The diff for Deisgner.cs files doesn't expand by default